packages/commonwealth/client/scripts/views/components/sidebar/sidebar_section.tsx

```diff
@@ -173,7 +173,7 @@ export const SidebarSectionGroup = (props: SidebarSectionAttrs) => {
 
     setToggled(!toggled);
 
-    localStorage.setItem(`${sectionName}-toggled`, (!!toggled).toString());
+    localStorage.setItem(`${sectionName}-toggled`, (!toggled).toString());
```

Comment:
This was the main issue here, it assumed that setToggled change the state of toggled when this code is ran, but in reality it changes the state of toggled when the component is refreshed, but when execution reaches this point, toggled has not been inverted from the setToggled.